mlrun/common/schemas/feature_store.py

```diff
@@ -27,42 +27,52 @@
 )
 
 
-class Feature(BaseModel):
+class FeatureStoreBaseModel(pydantic.BaseModel):
+    """
+    Intermediate base class, in order to override pydantic's configuration, as per
+    https://docs.pydantic.dev/1.10/usage/model_config/#change-behaviour-globally
+    """
+
+    class Config:
+        copy_on_model_validation = "none"
+
+
+class Feature(FeatureStoreBaseModel):
     name: str
     value_type: str
     labels: Optional[dict] = {}
 
     class Config:
-        extra = Extra.allow
+        extra = pydantic.Extra.allow
```

Comment:
Does this apply on top of the base-class? I mean, doesn't the internal class Config override the same class defined in the parent class? The example you point at have the class Config in the parent class, but not in the inherited classes. Possibly pydantic is doing some black magic allowing this, but have you verified this works?